src/main/java/net/sacredlabyrinth/phaed/simpleclans/managers/SettingsManager.java

```diff
@@ -813,69 +170,42 @@ public boolean hasDisallowedColor(String str) {
      * @return a comma delimited string with all disallowed colors
      */
     public String getDisallowedColorString() {
-        String out = "";
-
-        for (Object c : getDisallowedColors()) {
-            out += c + ", ";
-        }
-
-        return Helper.stripTrailing(out, ", ");
+        return String.join(", ", getStringList(DISALLOWED_TAG_COLORS));
     }
 
     /**
-     * Check whether a clan is un-rivable
+     * Check whether a clan is unrivable
      *
      * @param tag the tag
      * @return whether the clan is unrivable
      */
     public boolean isUnrivable(String tag) {
-        for (Object t : getunRivableClans()) {
-            if (((String) t).equalsIgnoreCase(tag)) {
-                return true;
-            }
-        }
-
-        return false;
-    }
-
-    @SuppressWarnings("deprecation")
-    public boolean isBanned(String playerName) {
-        return isBanned(Bukkit.getOfflinePlayer(playerName).getUniqueId());
+        return getStringList(UNRIVABLE_CLANS).contains(tag.toLowerCase());
```

Comment:
`contains` not the best choice here, should be `equalsIgnoreCase`.

Reply:
Or stream() and map to lower case

Reply:
> Or stream() and map to lower case

That was what I meant 👀